src/Auth0.OidcClient.Android/Auth0Client.cs

```diff
@@ -9,16 +12,98 @@ namespace Auth0.OidcClient
     public class Auth0Client : Auth0ClientBase
     {
         /// <summary>
-        /// Creates a new instance of the Auth0 OIDC Client.
+        /// Create a new instance of <see cref="Auth0Client"/> with a given <see cref="Auth0ClientOptions"/>.
         /// </summary>
-        /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration for the Auth0 OIDC Client.</param>
+        /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration to use.</param>
+        /// <remarks>options.RedirectUri must match your <see cref="Activity"/> <see cref="IntentFilterPriority"/>
+        /// DataScheme, DataPathPrefix and DataHost values.
+        /// If not supplied it will presume the convention
+        /// <code>$"{Context.PackageName}://{options.Domain}/android/{Context.PackageName}/callback".ToLower();</code>.
```

Comment:
Maybe just refer to `GetConventionCallbackUri` instead of duplicating that here

Reply:
GetConventionCallbackUri is private so will never show up in the intellisense.  If anything could remove it from GetConventionCallbackUri .